boards/riscv/titanium_ti60_f225/Kconfig.board

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2023 Efinix Inc.
```

Comment:
@manoj153 - can you verify that you are an Efinix employee? I just want to ensure that copyright is assigned properly here.

Reply:
Yes